File: app/src/main/java/com/example/breezil/favnews/di/module/SearchActivityModule.java
package com.example.breezil.favnews.di.module;

import com.example.breezil.favnews.ui.SearchActivity;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;


@Module
public abstract class SearchActivityModule {
    @ContributesAndroidInjector()
    abstract SearchActivity contributeSearchActivity();
}

File: app/src/main/java/com/example/breezil/fave/di/module/SearchActivityModule.kt
package com.example.breezil.fave.di.module

import com.example.breezil.fave.ui.explore.SearchActivity

import dagger.Module
import dagger.android.ContributesAndroidInjector


@Module
abstract class SearchActivityModule {
    @ContributesAndroidInjector
    internal abstract fun contributeSearchActivity(): SearchActivity
}